app/config/config_dev.yml

```diff
@@ -1,9 +1,6 @@
 imports:
    - { resource: config.yml }
 
-chameleon_system_debug:
```

Comment:
But `chameleon_system_debug: database_profiler: backtrace_enabled` is not added?

That is already false by default I guess?

Reply:
Yes.